service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/AbstractEmailContentProviderTest.java

```diff
@@ -35,7 +36,7 @@ protected String caseUrl(String caseId) {
         return formatCaseUrl(UI_URL, Long.valueOf(caseId));
     }
 
-    protected String caseUrl(String caseId, String tab) {
+    protected String caseUrl(String caseId, TabLabel tab) {
```

Comment:
not sure I like this method, we shouldn't be using production code in test methods

Reply:
also isn't this essentially repeating what is in the TestDataHelper?